

export const Genarate_Calulation= async (data1:any) => {
    try {
        const value = data1;
        return value
    
      } catch (e) {
        console.log('Failed to Genarate_Calulation')
      }

}